lib/model/prefs.js

```diff
@@ -77,7 +80,7 @@ const storePrefs = (changePrefEvent) => {
   } else if (changePrefEvent.name === 'server_changed') {
     storeServer(changePrefEvent.details);
   } else if (changePrefEvent.name === 'show_prefs') {
-    changePrefEvent = eventForShowPref(changePrefEvent);
+    return eventForShowPref(changePrefEvent);
```

Comment:
?

Reply:
The function `storePrefs` was modifying the input argument `changePrefEvent` storing and then returning it. Instead, the above modification returns the new value directly.